src/main/java/frc/robot/constants/RobotConstants.java

```diff
@@ -0,0 +1,13 @@
+package frc.robot.constants;
+/**
+ * Software/hardware constants (e.g. CAN IDs, gear ratios, field measurements, etc.). For software configs @see RobotConfig 
+ */
+public final class RobotConstants {
+
+    /**
+     * Drivetrain configs
+     */
+    public static final class DriveConstants {
```

Comment:
Delete this and it's associated docstring. This should be a part of the drivetrain commit.